src/routes/userRoutes.ts

```diff
@@ -1,5 +1,5 @@
 import express from "express"
-import { createUser, signIn, logout, updateUserProfile } from "../controllers/userController"
+import { createUser, signIn, logout, updateUserProfile, resetPasswordEmail, resetPassword } from "../controllers/userController"
```

Comment:
Can you add a comma here, so that the codes can be formatted easily and look cleaner? I mean after resetPassword

Reply:
what do you mean? You want it to look like this `"import { createUser, signIn, logout, updateUserProfile, resetPasswordEmail, resetPassword, } from "../controllers/userController"`